packages/proto/src/gen/babylon.btcstaking.v1.rs

```diff
@@ -345,56 +345,56 @@ pub struct NewFinalityProvider {
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct ActiveBtcDelegation {
     /// staker_addr is the address to receive rewards from BTC delegation.
-    #[prost(string, tag="11")]
+    #[prost(string, tag="1")]
```

Comment:
A lot of trouble caused by this. These tags are ids, and as such, they don't need to be sequential, start from 1, have a delta of 1, etc. They just have to be valid integers, and be unique in a given message.

For backwards compatibility reasons (and ease of integration), it makes much more sense to keep these tags unchanged. I know it looks "bad" or disorganised when these numbers are not sequential, but the truth is that keeping them unchanged wouldl make our lives easier.

Just mentioning this for awareness, perhaps at some point we can agree on this.

Reply:
In general I agree with this. Since phase-3 is still not live yet, we have the freedom of changing their values. For now we can try to make those clean. After the phase-3 testnet those values will not be changed unless there is really compelling reasons